priv/SimpleElection.aes

```diff
@@ -0,0 +1,455 @@
+@compiler >= 4.3
```

Comment:
This contract is already present in the repository under `contracts/Election.aes`. That file should be used instead of this one

Reply:
Generally, only .erl files should be present under /src
Am I to keep it under /src/contracts anyway?